### PR TITLE
Fixes flashlight eyes

### LINKED ISSUE #26413: If you set someone on fire their flashlight eyes stop working
@XDTM
@optimumtact 
I told you this was not a proper fix, and, surprise, it has caused a bug.

### PR DESCRIPTION
**Pull request by QualityVan**
Fixes #26413, flashlight eyes now contain actual flashlights. Can't toggle them while they're inside you, though.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
you appear to have nerfed flashlight eyes!
would you like to try again?

File: code/modules/surgery/organs/organ_internal.dm [hunk 1]
```diff
@@ -878,17 +878,33 @@
 	icon_state = "flashlight_eyes"
 	flash_protect = 2
 	tint = INFINITY
+	var/obj/item/device/flashlight/eyelight/left_eye
+	var/obj/item/device/flashlight/eyelight/right_eye
```
Comment (ChangelingRain) [inline code/modules/surgery/organs/organ_internal.dm]: You don't need two eyes here.

**Comment by QualityVan** [comment]
I'm not sure I follow? Unless you mean the old version previously using 15 for its range and 1 for its power, in which case oops. Not entirely sure how to make two lights be as strong as a single one, so maybe just cut the pair?

**Comment by octareenroon91** [comment]
Maybe just drive this with one flashlight

File: code/modules/surgery/organs/organ_internal.dm [hunk 2]
```diff
@@ -878,17 +878,24 @@
...
+	eye.on = TRUE
+	eye.loc = M
+	eye.update_brightness(M)
 
 
 /obj/item/organ/eyes/robotic/flashlight/Remove(var/mob/living/carbon/M, var/special = 0)
-	M.set_light(M.light_range -15, M.light_power - 1)
+	eye.on = FALSE
+	eye.update_brightness(M)
+	eye.loc = src
```
Comment (ChangelingRain) [inline code/modules/surgery/organs/organ_internal.dm]: eye.forceMove(src)

File: code/modules/surgery/organs/organ_internal.dm [hunk 3]
```diff
@@ -878,17 +878,24 @@
...
 /obj/item/organ/eyes/robotic/flashlight/emp_act(severity)
 	return
 
 /obj/item/organ/eyes/robotic/flashlight/Insert(var/mob/living/carbon/M, var/special = 0)
 	..()
-	M.set_light(M.light_range + 15, M.light_power + 1)
+	if(!eye)
+		eye = new /obj/item/device/flashlight/eyelight()
+	eye.on = TRUE
+	eye.loc = M
```
Comment (ChangelingRain) [inline code/modules/surgery/organs/organ_internal.dm]: eye.forceMove(M)

File: code/game/objects/items/devices/flashlight.dm [hunk 4]
```diff
@@ -428,3 +428,11 @@
...
 	brightness_on = 2.5
 	flashlight_power = -3
+
+/obj/item/device/flashlight/eyelight
+	name = "eyelight"
+	desc = "This shouldn't exist outside of someone's head, how are you seeing this?"
+	brightness_on = 15
+	flashlight_power = 1
+	flags = CONDUCT | DROPDEL
+	actions_types = list()
```
Comment (coiax) [inline code/game/objects/items/devices/flashlight.dm]: This is a hack, we *really* should just make organs inside the person. Not your fault.

**Comment by QualityVan** [comment]
holy shit that's not how rebasing works
let's try this again

**Comment by QualityVan** [comment]
Okay I think it worked this time

**Pull request closed without merging**